### ISSUE TITLE
Thank you Kin Lane

### ISSUE THREAD

**Issue by ammtoo** [swagger-api/swagger-ui#8271]
I recently learned that Kin Lane "was involved in the whole journey, helping build up and popularize Swagger, as well as the transition from Swagger to OpenAPI": https://apievangelist.com/2022/06/25/swagger-reflects-the-short-signtedness-of-many-api-industry-service-providers/

I think there many hidden heroes behind Swagger who deserve more recognition of their contributions.

Thank you Kin. Without you, we won't have awesome tools like swagger-ui.

**Comment by ch1917** [comment]
Kudos to his contributions. He plays a significant role in where the Swagger brand is today: https://apievangelist.com/2015/10/22/how-to-build-an-api-brand-through-consistent-storytelling/

There are definitely something we can learn from his contributions. When I searched for ["org:swagger-api author:kinlane"](https://github.com/search?q=org%3Aswagger-api+author%3Akinlane), it didn't return any commits authored by "kinlane". Tried ["org:swagger-api author:apievangelist"](https://github.com/search?q=org%3Aswagger-api+author%3Aapievangelist) too but got nothing

Does anyone know where I can find his contributions (commits) to Swagger so that I can learn from his code? Thanks.

**Comment by bytetob** [comment]
Yup, definitely want to learn from him as he's fluent in JavaScript: https://apievangelist.com/2017/10/23/a-simple-api-with-aws-dynamodb-lambda-and-api-gateway/

**Comment by triplerocket** [comment]
> There are definitely something we can learn from his contributions. When I searched for ["org:swagger-api author:kinlane"](https://github.com/search?q=org%3Aswagger-api+author%3Akinlane), it didn't return any commits authored by "kinlane". Tried ["org:swagger-api author:apievangelist"](https://github.com/search?q=org%3Aswagger-api+author%3Aapievangelist) too but got nothing

The author of https://whoisanexpert.github.io/learn/api/2022/09/10/a-few-things-to-learn-from-an-api-expert.html notices the same - Kin's contributions to Swagger projects can not be found at all.

I learn from this discussion in Reddit (https://www.reddit.com/r/AskProgramming/comments/xd5bj1/programming_language_fluency/) that he claims to be fluent in PHP but some people disagree by reviewing his PHP projects.

Does SmartBear or owner of https://github.com/swagger-api have more information regarding Kin's contributions to Swagger?

**Comment by orubel** [comment]
Well I believe Kin Lane credits his work on Swagger to Owen Rubel https://www.flickr.com/photos/orubel/51394882216/in/dateposted-public/

**Comment by marccucurella** [comment]
Mike Ralphson (OpenAPI Technical Steering Committee) has an answer for that: 

> I'm sorry to say I don't think @kinlane last long at @Bloomberg, he has always been a chancer riding on the coattails of others. He has zero commits in RAML, API Blueprint, Swagger or OpenAPI for example

Looks like Kin Lane (@kinlane) has been a free rider for a while.

Ref: https://archive.is/WHMoe
